<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="editorHelp_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="editorConsole_splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="documents_tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>5</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Python Console</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="pyConsole_output">
              <property name="font">
               <font>
                <family>Courier</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="lineWrapMode">
               <enum>QTextEdit::NoWrap</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextBrowserInteraction</set>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
              <property name="openLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="pyconsole_prompt_layout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="pyConsole_prompt">
                <property name="font">
                 <font>
                  <family>Courier</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: white;</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>&gt;&gt;&gt;</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::PlainText</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Run Script</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Run Command</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="runScript_command">
              <property name="font">
               <font>
                <family>Courier</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="runScript_button">
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="runScript_output">
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTextEdit" name="helpPane">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_File_As"/>
    <addaction name="actionPrint"/>
    <addaction name="actionClose_Tab"/>
    <addaction name="actionClose_All_Tabs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_and_Replace"/>
   </widget>
   <widget class="QMenu" name="menuShell">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_Current_Script"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFIDLE_Help"/>
    <addaction name="actionPython_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_fIDEL"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuShell"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_File_As">
   <property name="text">
    <string>Save File As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
  <action name="actionClose_All_Tabs">
   <property name="text">
    <string>Close All Tabs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_and_Replace">
   <property name="text">
    <string>Find and Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart Python Console</string>
   </property>
  </action>
  <action name="actionFIDLE_Help">
   <property name="text">
    <string>fIDLE Help</string>
   </property>
  </action>
  <action name="actionPython_Help">
   <property name="text">
    <string>Python Help</string>
   </property>
  </action>
  <action name="actionAbout_fIDEL">
   <property name="text">
    <string>About fIDLE</string>
   </property>
  </action>
  <action name="actionRun_Current_Script">
   <property name="text">
    <string>Run Current Script</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>documents_tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>close_tab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>documents_tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>handle_tab_change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>close_tab(int)</slot>
  <slot>send_pyconsole_command()</slot>
  <slot>handle_tab_change(int)</slot>
 </slots>
</ui>
